Gemfile

```diff
@@ -19,4 +19,6 @@ gem("tapioca", require: false)
 
 group :development do
   gem("byebug", require: false)
+  gem("minitest-focus", require: false)
```

Comment:
The minitest-focus gem is being introduced in https://github.com/Shopify/packwerk/pull/40